open-build-service-api/src/lib.rs

```diff
@@ -87,7 +87,12 @@ impl PackageCode {
     pub fn is_final(&self) -> bool {
         matches!(
             self,
-            Self::Disabled | Self::Succeeded | Self::Failed | Self::Excluded
+            Self::Broken
```

Comment:
Reasonable; but we need to be a bit careful;  I'm sure i've seen obs report broken temporarily until it reevaluate what was pushed (though i could be misremembering).

Reply:
Hmm, ditto for this one being taken from ci-package-builder too, I'm not sure if there would be a particular issue in that case too?